src/hotspot/share/utilities/defaultStream.hpp

```diff
@@ -30,6 +30,7 @@
 class defaultStream : public xmlTextStream {
   friend void ostream_abort();
  public:
+  class TestSupport;  // Unit test support
```

Comment:
Because I use `tty` which is a `defaultStream` so that is where the test code has to be placed to access the protected function.

BTW this pattern was suggested by Kim Barrett in preference to the use of a friend class.